packages/visualizations/addon/templates/components/cell-renderers/metric.hbs

```diff
@@ -1,2 +1,2 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{metricValue}}
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{format-table-cell metricValue format}}
```

Comment:
Do we need this helper for a single use? Can we bring this into the metric cell-renderer

Reply:
yeah. It seems that it is not needed anymore

Reply:
@jkusa removed